server/controller.js

```diff
@@ -15,6 +16,88 @@ const controller = {
           res.status(404).send(err);
         });
     },
+    // POST
+    post: (req, res) => {
+      // validate request
+      if (!req.params.itemId) {
+        res.status(400).send({
+          message: 'Review cannot be created!',
+        });
+        return;
+      }
+      // create new review
+      const newReview = {
+        customer_name: req.body.customer_name,
+        date_of_review: req.body.date_of_review,
+        rating: req.body.rating,
+        review_content: req.body.review_content,
+        image_url: req.body.image_url,
+        item_option: req.body.item_option,
+        ItemId: req.params.itemId,
+      };
+      Review.create(newReview)
```

Comment:
Following the MVC design paradigm, the creation of the review in the database should be inside of a model folder / file. I think its ok to create the actual review object (newReview) inside the controller and pass that as an argument to a function from the model that will do the actual "posting" of the review.

This approach should be used for all of the CRUD endpoints if you are trying to achieve separation of concerns.